Doc/Tutorial/chapter_align.tex

```diff
@@ -975,9 +975,122 @@ \section{Reading and writing alignments}
 \subsection{Reading alignments}
 \label{subsec:align_reading}
 
-The alignments iterator returned by \verb|Bio.Align.parse| inherits from the \verb|AlignmentsAbstractBaseClass| base class. This class defines a \verb|rewind| method that resets the iterator to let it loop over the alignments from the beginning. You can also call \verb|len| on the alignments to obtain the number of alignments. Depending on the file format, the number of alignments may be explicitly stored in the file (for example in the case of bigBed, bigPsl, and bigMaf files), or otherwise the number of alignments is counted by looping over them once (and returning the iterator to its original position). If the file is large, it may therefore take a considerable amount of time for \verb|len| to return. As the number of alignments is cached, subsequent calls to \verb|len| will return quickly.
+Use \verb|Bio.Align.parse| to parse a file of sequence alignments. For example, the file \verb|ucsc_mm9_chr10.maf| contains 48 multiple sequence alignments in the MAF (Multiple Alignment Format) format (see section \ref{subsec:align_maf}):
+
+%cont-doctest
```

Comment:
This looks like it could/should be the start of a new self-contained doctest?

Reply:
It is a continuation, otherwise we'd have to redefine a bunch of variables in the next cont-doctest block.

Reply:
OK - that's a shame. At some point a fresh start becomes helpful for readers too as they may be dipping into the examples rather than following an entire section.